libs/BehaviorKit/include/BehaviorKit.h

```diff
@@ -4,16 +4,16 @@
 
 #pragma once
 
-#include "LedKit.h"
 #include "interface/drivers/Motor.h"
+#include "interface/libs/LedKit.h"
 #include "interface/libs/VideoKit.h"
 
 namespace leka {
 
 class BehaviorKit
 {
   public:
-	explicit BehaviorKit(interface::VideoKit &videokit, LedKit &ledkit, interface::Motor &motor_left,
+	explicit BehaviorKit(interface::VideoKit &videokit, interface::LedKit &ledkit, interface::Motor &motor_left,
```

Comment:
not related to the PR, but could behaviors depend on motionkit instead of motors? how are motors used here?